open-reac/src/test/java/com/powsybl/openreac/parameters/input/VoltageLevelLimitsOverrideInputTest.java

```diff
@@ -190,6 +190,44 @@ void testVoltageOverrideWithLowLimitGreaterHighLimit() {
         assertEquals("Override on voltage level " + vl.getId() + " leads to low voltage limit >= high voltage limit.", e3.getMessage());
     }
 
+    @Test
+    void testVoltageOverrideWithLowLimitGreaterNominalVoltage() {
+        Network network = IeeeCdfNetworkFactory.create57();
+        setDefaultVoltageLimits(network); // set default voltage limits to every voltage levels of the network
+
+        VoltageLevel vl = network.getVoltageLevels().iterator().next();
+        vl.setHighVoltageLimit(480);
+        vl.setLowVoltageLimit(400);
```

Comment:
Is it normal to have inconsistent values, even before overriding them? Here, we already have the low voltage limit greater than the nominal V.

Reply:
Changed